daemon/call_interfaces.c

```diff
@@ -1455,7 +1455,7 @@ static const char *call_offer_answer_ng(struct ng_buffer *ngbuf, bencode_item_t
 	detect_setup_recording(call, &flags.record_call_str, &flags.metadata);
 	if (flags.record_call) {
 		call->recording_on = 1;
-		recording_start(call, NULL, &flags.metadata);
+		recording_start(call, NULL, &flags.metadata, NULL);
```

Comment:
There's a chance here that this new parameter can be used in other messages also, e.g. in offer/answer. The parsing of the dict key could be offloaded to call_ng_process_flags() and stored in `flags` - just a suggestion